ppr-ui/src/utils/ltsa-api-helper.ts

```diff
@@ -0,0 +1,32 @@
+// Libraries
+import { axios } from '@/utils/axios-ppr'
+import { StatusCodes } from 'http-status-codes'
+import { SearchResponseIF } from '@/interfaces'
+import { ErrorCategories } from '@/enums'
+
+// Submit an LTSA summary query request.
+// Testing PID #: 014597365
+export async function ltsaSummary (
+  pidNumber: string
+): Promise<any> {
+  const url = sessionStorage.getItem('LTSA_API_URL')
+  const config = { baseURL: url, headers: { Accept: 'application/json' } }
+
+  return axios
+    .get<SearchResponseIF>(`titledirect/search/api/titleSummaries?filter=parcelIdentifier:${pidNumber}`, config)
+    .then(response => {
+      const data = response?.data
+      if (!data) {
+        throw new Error('Invalid API response')
+      }
+      return data
+    })
+    .catch(error => {
+      return {
+        error: {
+          category: ErrorCategories.LTSA_REQUEST,
```

Comment:
Are we actually doing anything with error details? Do we have some built-in handling (i.e. log entry, show on screen, console etc.)?

Reply:
Nope, not with these values directly, in this context.
It could be useful if we wanted to pass the statusCode and/or message to an error dialog. 

In this case, we are simply prompting a generic dialog, design doesn't call for specific messaging. 
I have to imagine this IS happening in other scenarios however, since this is the approach that is more or less consistent with other request helpers.